src/components/scrollable-canvas/scrollable-canvas.css

```diff
@@ -10,17 +10,17 @@ $scrollbar-padding: 1px;
 .vertical-scrollbar-wrapper {
     position: absolute;
     width: $scrollbar-size;
-    right: 0;
+    right: $scrollbar-padding;
```

Comment:
@adroitwhiz brings up this issue that it would be helpful if the padding counted as additional hitbox: https://github.com/LLK/scratch-paint/issues/1011